depot/utils.py

```diff
@@ -1,8 +1,13 @@
-from unidecode import unidecode
 from ._compat import percent_encode
 
+try:
+    from unidecode import unidecode as fix_chars
+except ImportError:
+    from unicodedata import normalize
+    def fix_chars(string):
+        return percent_encode(normalize('NFKC', string), safe='!#$&+-.^_`|~', encoding='utf-8')
```

Comment:
Percent encoding doesn't seem to achieve the goal of this which was mostly to ensure that a "plain ascii version" of filenames was provided in download such that the file could be saved under any kind of OS/filesystem.

Maybe evaluate replacing unidecode with anyascii ( https://github.com/hunterwb/any-ascii ) that should be under a compatible license ( https://github.com/hunterwb/any-ascii/blob/master/LICENSE ) and leads to nearly equivalent results.

```
>>> from anyascii import anyascii
>>> 
>>> anyascii("Крупный")
'Krupnyy'
>>> anyascii("àèìòù")
'aeiou'
```